packages/aws-rfdk/lib/lambdas/lambda-layer-version-arn-mapping.ts

```diff
@@ -33,11 +33,25 @@ type LambdaLayerVersionArnMap<T> = { [layer: string]: T };
  * A `CfnMapping` of our lambda layer version ARNs, from the autogenerated `lambdaLayerVersionArns.ts` file.
  */
 export class LambdaLayerVersionArnMapping extends CfnMapping {
+  /**
+   * Gets a Lambda LayerVersion for the specified layer.
+   *
+   * Note that this uses a `CfnMapping` that is unique within a stack. This method will create a `CfnMapping`
+   * if it does not already exist (i.e. this method has not been called yet).
+   * @param scope The scope to create the LayerVersion in.
+   * @param id The ID of the construct.
+   * @param layer The layer to get.
+   */
+  public static getLambdaLayerVersion(scope: Construct, id: string, layer: LambdaLayer): ILayerVersion {
+    const instance = this.getSingletonInstance(scope);
+    return LayerVersion.fromLayerVersionArn(scope, id, instance.findInMap(instance.stack.region, layer));
+  }
+
   /**
    * Gets the singleton instance (per-stack) of this class.
    * @param scope The parent construct.
    */
-  public static getSingletonInstance(scope: Construct) {
+  protected static getSingletonInstance(scope: Construct) {
```

Comment:
Shouldn't this be private?

Reply:
Yeah, no need for it to be `protected` yet, changed